RUFAS/routines/animal/manure/lactating_cow_manure_excretion.py

```diff
@@ -6,135 +6,201 @@
-        bw: body weight, kg
-        days_milk: days in milk, d
-        milk_protein: milk protein, % of milk (from animal input)
-        p_feces_excrt: amount of P excreted by an animal (g)
-        p_urine: amount of P required for urine production (g)
-        methane_model: methane model used for methane emission calculations
-        milk_fat: milk fat, % of milk
-        ME_intake: metabolizable energy intake, Mcal/kg DM
-
-    Returns:
-        p_excrt: amount of P excreted by animal, g
-        and a dictionary containing the following values
-            U: urea concentration, mol/L
-            Urine: # TODO: Add description
-            TAN_s: total ammoniacal nitrogen concentration in the manure slurry,
-                mol/L
-            MN: nitrogen in liquid and solid manure, g
-            Mkg: amount of manure, kg
-            VSd: degradable volatile solids, g
-            VSnd: non-degradable volatile solids, g
-            WIP_frac: water extractable inorganic P fraction
-            WOP_frac: water extractable organic P fraction
-            p_excrt_manure: manure P excretion for manure module input (g)
-            p_frac: P fraction of manure
-            K: potassium in manure, g/day
+def manure_calculations(ration_formulation,
+                        feed,
+                        body_weight: float,
+                        days_in_milk: int,
+                        milk_protein: float,
+                        daily_milk_production: float,
+                        fecal_phosphorus: float,
+                        urine_phosphorus_required: float,
+                        methane_model: str,
+                        milk_fat: float,
+                        metabolizable_energy_intake: float) \
+        -> Tuple[float, AnimalManureExcretions]:
+    """Calculates the manure excretion values for a cow with information from the ration formulation.
+
+    Parameters
+    ----------
+    ration_formulation : Dict[str, float]
+        Dictionary that stores the calculated ration.
+    feed : Dict[str, float]
+        A Feed object that contains information about the available feeds.
+    body_weight : float
+        Body weight of the current cow, kg.
+    days_in_milk : int
+        Days in milk, days.
+    milk_protein : float
+        Milk protein (from animal input), % of milk.
+    daily_milk_production : float
+        Daily milk production of the current cow, kg.
+    fecal_phosphorus : float
+        Amount of fecal phosphorus excreted by the current cow, g.
+    urine_phosphorus_required : float
+        Amount of phosphorus required for urine production, g.
+    methane_model : str
+        Methane model used for methane emission calculations, including Mutian, Mills, IPCC.
+    milk_fat : float
+        Milk fat (from animal input), % of milk.
+    metabolizable_energy_intake : float
+        Metabolizable energy intake, Mcal/kg dry matter.
+
+    Returns
+    -------
+    float
```

Comment:
Similar to other comments, do the returns need to have their variable names included?
```suggestion
    total_phosphorus_excreted : float
```